lib/matplotlib/axes/_axes.py

```diff
@@ -6104,8 +6104,15 @@ def pcolorfast(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
 
         .. warning::
 
-           This method is experimental. Compared to `~.Axes.pcolor` or
-           `~.Axes.pcolormesh` it has some limitations:
+        C can in some cases be 3D with the last dimension as rgb(a).
+        This is available when C qualifies for image or pcolorimage type,
+        otherwise if C is 3D, a TypeError will be raised.
```

Comment:
I think this should raise a ValueError instead (the types are all correct).

Reply:
I guess that might be better. If others agree I can change this.

Reply:
Changed